CONTRIBUTING.md

```diff
@@ -80,27 +70,35 @@ Must be one of the following:
 * **style**: Changes that do not affect the meaning of the code (white-space, formatting, etc)
 * **refactor**: A code change that neither fixes a bug nor adds a feature
 * **perf**: A code change that improves performance
+* **revert**: Reverts a previous commit
 * **test**: Adding missing tests
 * **chore**: Changes to the build process or auxiliary tools and libraries such as documentation
   generation
 
-### Scope
+##### Revert
+If the commit reverts a previous commit, it should begin with `revert: `, followed by the header of the reverted commit. In the body it should say: `Revert commit <hash>.`, where the hash is the SHA of the commit being reverted.
+
+#### Scope
 The scope could be anything specifying place of the commit change. For example `$location`,
-`$browser`, `$compile`, `$rootScope`, `ngHref`, `ngClick`, `ngView`, etc...
+`$browser`, `$compile`, `$rootScope`, `ngHref`, `ngClick`, `ngView`, etc.
 
-### Subject
+#### Subject
 The subject contains succinct description of the change:
 
 * use the imperative, present tense: "change" not "changed" nor "changes"
 * don't capitalize first letter
 * no dot (.) at the end
 
+A properly formed git commit subject line should always be able to complete the following sentence:
+
+> If applied, this commit will ___your subject line here___
+
 ### Body
-Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes".
+Wrap the body at 72 characters whenever possible (for example, don't modify long links to follow this rule). Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes".
 The body should include the motivation for the change and contrast this with previous behavior.
 
-### Footer
-The footer should contain any information about **Breaking Changes** and is also the place to
-reference GitHub issues that this commit **Closes**.
+The body contains (in order of appearance) a detailed **description** of the committed changes, references GitHub issues that the commit **closes** and any **breaking changes**. The **description** is mandatory, **closes** and **breaking changes** are optional.
+
+GitHub issues that the commit **closes** should start with the word `Closes` or `Fixes`.
```

Comment:
>Do you think this is not clear enough?

I think that it's wrong to claim that GH requires capital letters, since this is not the case.

Also, now that I look at it, whole sentence is kinda ambiguous, and confusing even when considering the sentence before.